### PR TITLE
Feat: Added Card and Image component

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added various Card
   -Cards with badges
   -Cards with dismiss
   -Cards with text overlay
   -Text only cards
   -Horizontal cards
   -Cards with shadow
   
  Added Images
     -Responsive images
     -Rounded Images

### CODE REVIEW

File: components/CSS/card.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}
+/* @media (min-width: 800px) {
+  .card-layout {
+    justify-content: center;
+  }
+} */
+
+.wrapper-card {
+    display: flex;
+    font-family: --font-playfair;
```
Comment (raieegulhane) [inline components/CSS/card.css:12]: var(--font-playfair) needed.

File: components/CSS/card.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    flex-direction: column;
+    padding: var(--size-one) 8px;
+    border: 1px solid gray;
+    align-items: center;
+    box-shadow: 0 0 5px 5px lightgray;
```
Comment (raieegulhane) [inline components/CSS/card.css:17]: Use variables for color

**Review by raieegulhane — COMMENTED** [review]
Please check comments. Rest is good to go.

**Review by vikrantkode — COMMENTED** [review]
Good to go. Can change the suggested change

**Comment by anurag-kkumarr** [comment]
Thanks for the PR review

**Pull request merged**